WalletWasabi/Wallets/WalletFilterProcessor.cs

```diff
@@ -45,44 +49,43 @@ public WalletFilterProcessor(KeyManager keyManager, BitcoinStore bitcoinStore, T
 		BlockProvider = blockProvider;
 	}
 
-	/// <remarks>Guarded by <see cref="SynchronizationRequestsLock"/>.</remarks>
+	/// <remarks>Guarded by <see cref="Lock"/>.</remarks>
 	private PriorityQueue<SyncRequest, Priority> SynchronizationRequests { get; } = new(Comparer);
-	
+
 	/// <remarks>Guards <see cref="SynchronizationRequests"/>.</remarks>
-	private object SynchronizationRequestsLock { get; } = new();
+	private object Lock { get; } = new();
 	private SemaphoreSlim SynchronizationRequestsSemaphore { get; } = new(initialCount: 0);
 
 	private KeyManager KeyManager { get; }
 	private BitcoinStore BitcoinStore { get; }
 	private TransactionProcessor TransactionProcessor { get; }
 	private IBlockProvider BlockProvider { get; }
 	public FilterModel? LastProcessedFilter { get; private set; }
-	private Dictionary<uint, FilterModel> FiltersCache { get; } = new ();
-	
-	/// <summary>Make sure we don't process any request while a reorg is happening.</summary>
-	private AsyncLock ReorgLock { get; } = new ();
 
-	private void Add(SyncRequest request)
-	{
-		lock (SynchronizationRequestsLock)
-		{
-			Priority priority = new(request.SyncType);
-			SynchronizationRequests.Enqueue(request, priority);
-			SynchronizationRequestsSemaphore.Release(releaseCount: 1);
-		}
-	}
+	/// <remarks>Internal only to allow modifications in tests.</remarks>
+	internal Dictionary<uint, FilterModel> FiltersCache { get; } = new();
```

Comment:
We only use internal that way, so the comment is useless. Feel free to keep it if you think it's right.